test/verify/machineslib.py

```diff
@@ -1365,6 +1375,7 @@ class TestCreateConfig:
         VALID_DISK_IMAGE_PATH = '/var/lib/libvirt/images/example.img'
         NOVELL_MOCKUP_ISO_PATH = '/var/lib/libvirt/novell.iso'  # libvirt in ubuntu-1604 does not accept /tmp
         NOT_EXISTENT_PATH = '/tmp/not-existent.iso'
+        VALID_DIRECT_ISO_URL = 'http://download.eng.brq.redhat.com/pub/fedora/linux/releases/29/Server/x86_64/iso/Fedora-Server-dvd-x86_64-29-1.2.iso'
```

Comment:
This does not work. Our tests don't have network access, and even more so, this is behind the Red Hat internal VPN. The test could use [python3 -m httpserver](https://docs.python.org/3/library/http.server.html) to serve the local CirrOS image and test against that?